### ISSUE TITLE
How to install the Chinese version?

### ISSUE THREAD

**Issue by iwchang** [Proektsoftbg/Calcpad#406]
How to install the Chinese version? I don't know how to import the Resources folder under the Translation project, and there is no guidance inside. And why not provide a setting function to facilitate language settings?

**Comment by Proektsoftbg** [comment]
Hi!

Only the old version is available in Chinese:

[calcpad-setup-zh-x64.exe](https://github.com/Proektsoftbg/Calcpad/blob/main/Setup/Calcpad%206.2.3/calcpad-setup-zh-x64.exe)
[calcpad-setup-zh-x64.zip](https://github.com/Proektsoftbg/Calcpad/blob/main/Setup/Calcpad%206.2.3/calcpad-setup-zh-x64.zip)

The new version is not translated yet. @idealkindom was about to do it, but I do not know what is the current progress. Maybe he is too busy. I think it would be great if more people  with Chinese join and help him for this job. I cannot do it, Chinese is a very difficult language and I cannot get anything.

**Comment by idealkindom** [comment]
> Hi!
> 
> Only the old version is available in Chinese:
> 
> [calcpad-setup-zh-x64.exe](https://github.com/Proektsoftbg/Calcpad/blob/main/Setup/Calcpad%206.2.3/calcpad-setup-zh-x64.exe) [calcpad-setup-zh-x64.zip](https://github.com/Proektsoftbg/Calcpad/blob/main/Setup/Calcpad%206.2.3/calcpad-setup-zh-x64.zip)
> 
> The new version is not translated yet. @idealkindom was about to do it, but I do not know what is the current progress. Maybe he is too busy. I think it would be great if more people with Chinese join and help him for this job. I cannot do it, Chinese is a very difficult language and I cannot get anything.

Yes, I have been working on it. Hopefully, I will pull the request before Christmas.

Recently, I have been busy on working and baby caring.

**Comment by Proektsoftbg** [comment]
@idealkindom Great! Thank you! Whenever you manage. 
And also congrats for your baby. I wish health and happiness to all of you.

**Comment by idealkindom** [comment]
> @idealkindom Great! Thank you! Whenever you manage. And also congrats for your baby. I wish health and happiness to all of you.

Thks a lot and everything goes fine. The Trans work is more than 80% done.

**Comment by Proektsoftbg** [comment]
Hi! We are very close to having a Chinese VM 7.1.5 version, thanks to @idealkindom.
Some strings still need translation in resource files. I exported all of them to an excel file:
https://github.com/Proektsoftbg/Calcpad/blob/main/Translation/String%20tables%20VM%207.1.5.xlsx

If it is easier to translate there, you can do it and I can import them back.
About help files and Html templates, I added all CSS formatting and other things needed, so I think they are ready.

**Comment by idealkindom** [comment]
> Hi! We are very close to having a Chinese VM 7.1.5 version, thanks to @idealkindom. Some strings still need translation in resource files. I exported all of them to an excel file: https://github.com/Proektsoftbg/Calcpad/blob/main/Translation/String%20tables%20VM%207.1.5.xlsx
> 
> If it is easier to translate there, you can do it and I can import them back. About help files and Html templates, I added all CSS formatting and other things needed, so I think they are ready.

I will finish it as soon as possilbe.

**Comment by Proektsoftbg** [comment]
Hi! Take your time. There is nothing to hurry for. :) And thank you so much again for your help. You made a really great contribution to this project.

**Comment by idealkindom** [comment]
> Hi! Take your time. There is nothing to hurry for. :) And thank you so much again for your help. You made a really great contribution to this project.

Hey,you exaggerated my effect.I am just a bug explorer and adviser for ＃input command, but also I benefited from this project a lot.

**Comment by idealkindom** [comment]
> [@idealkindom](https://github.com/idealkindom) I published the Chinese version. https://github.com/Proektsoftbg/Calcpad/releases/tag/v7.1.6
> 
> Please, check if it is OK. Some things I got from your resx files here: https://github.com/Proektsoftbg/Calcpad/tree/main/Translation/Chinese/Resources Some I took from the Excel file: https://github.com/Proektsoftbg/Calcpad/blob/main/Translation/String%20tables%20VM%207.1.6.xlsx
> 
> For some strings, puctuation was changed from dot . to exclamation !. It gave and error there, so I changed it. Maybe this was wrong, but if it is not very incorrect in Chinese, I prefer to keep it. Please advice.
> 
> There were a few translations missing from Cli's Messages.zh.resx. I translated them with Google Translation. Maybe they are awful. Please check.
> 
> All translated files were included into the main code base. If you work with Visual Studio, it is best to install the ResX Manager addon and work on them directly if needed.

That's the best news I received on 2025.I used resXman, if I don't make an error of the software, which is also a opensource program on github.
Surely, I will test soon and see through it and give feedback when there is a need to.

**Comment by idealkindom** [comment]
> [@idealkindom](https://github.com/idealkindom) I published the Chinese version. https://github.com/Proektsoftbg/Calcpad/releases/tag/v7.1.6
> 
> Please, check if it is OK. Some things I got from your resx files here: https://github.com/Proektsoftbg/Calcpad/tree/main/Translation/Chinese/Resources Some I took from the Excel file: https://github.com/Proektsoftbg/Calcpad/blob/main/Translation/String%20tables%20VM%207.1.6.xlsx
> 
> For some strings, puctuation was changed from dot . to exclamation !. It gave and error there, so I changed it. Maybe this was wrong, but if it is not very incorrect in Chinese, I prefer to keep it. Please advice.
> 
> There were a few translations missing from Cli's Messages.zh.resx. I translated them with Google Translation. Maybe they are awful. Please check.
> 
> All translated files were included into the main code base. If you work with Visual Studio, it is best to install the ResX Manager addon and work on them directly if needed.

That's the best news I received on 2025.I used resXman, if I don't make an error of the software, which is also a opensource program on github.
Surely, I will test soon and see through it and give feedback when there is a need to.
